docs/topics/api/categories.rst

```diff
@@ -0,0 +1,125 @@
+==========
+Categories
+==========
+
+.. note::
+
+    These APIs are experimental and are currently being worked on. Endpoints
+    may change without warning.
+
+.. _category-list:
+
+Categories are defined by a name, a slug, a type and an application. Slugs are
```

Comment:
Not sure what our editorial policy on serial commas are, but I find lists like this read better with them in eg:

`a name, a slug, a type, and an application`.